packages/client/test/end-to-end/publish-subscribe-via-proxy.test.ts

```diff
@@ -1,17 +1,17 @@
 import { Wallet } from '@ethersproject/wallet'
-import { fastWallet, fetchPrivateKeyWithGas } from '@streamr/test-utils'
+import { describeOnlyInNodeJs, fastWallet, fetchPrivateKeyWithGas } from '@streamr/test-utils'
 import { ProxyDirection } from '@streamr/trackerless-network'
 import { collect, wait, withTimeout } from '@streamr/utils'
 import { Stream } from '../../src/Stream'
 import { StreamrClient } from '../../src/StreamrClient'
 import { StreamPermission } from '../../src/permission'
 import { createTestClient, createTestStream } from '../test-utils/utils'
 
-const TIMEOUT = 15 * 1000
+const TIMEOUT = 30 * 1000
 const SUBSCRIBE_WAIT_TIME = 2000
 const WEBSOCKET_PORT = 14231
 
-describe('publish/subscribe via proxy', () => {
+describeOnlyInNodeJs('publish/subscribe via proxy', () => { // Cannot run proxy server in browser
```

Comment:
Why is it not possible to run the proxy server in the electron? The `preload.js` in `browser-test-runner` it is possible to run WS servers in electron. (ConfigTest.ts enables WS servers for all other end-to-end test anyhow)

Reply:
Is there a trick to get it working? When I enable the test I get the following errors:
```
1) publish
     publish/subscribe via proxy
     Error: Timeout - Async function did not complete within 30000ms (custom timeout)
    at <Jasmine>

2) subscribe
     publish/subscribe via proxy
     Error: Timeout - Async function did not complete within 30000ms (custom timeout)
    at <Jasmine>
     TypeError: Cannot read properties of undefined (reading 'content')
    at UserContext.<anonymous> (/tmp/_karma_webpack_25212/commons.js:69299:32)
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (/tmp/_karma_webpack_25212/commons.js:69252:103)
    at _next (/tmp/_karma_webpack_25212/commons.js:69253:194)
```

Reply:
Yup, I have it a closer look and realised that the client webpack always starts the node in browser node. This means that starting the WS server is disabled even in Electron. Probably best to keep the test disabled in the browser then.

Reply:
The test could pass if we allow proxy connections to be formed with webrtc. (Requires the proxied client to join the DHT temporarily or at least to open a connection to an entry point to forward signaling messages)

Reply:
Maybe add a TODO to enable this test in a separate PR